src/components/Common/FilePreviewDialog.tsx

```diff
@@ -79,9 +101,23 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
     });
   };
 
+  const handleNext = (newIndex: number) => {
+    if (!uploadedFiles?.length) return;
```

Comment:
Let's combine these checks as well. Other than that, good to go.

Reply:
done